Sources/ViewInspector/Inspector.swift

```diff
@@ -116,19 +116,10 @@ private extension String {
     func replacingGenericParameters(_ replacement: String) -> String {
         guard let start = self.firstIndex(of: "<")
         else { return self }
-        var balance = 1
-        var current = self.index(after: start)
-        while balance > 0 && current < endIndex {
-            let char = self[current]
-            if char == "<" { balance += 1 }
-            if char == ">" { balance -= 1 }
-            current = self.index(after: current)
-        }
-        if balance == 0 {
-            return String(self[..<start]) + replacement +
```

Comment:
The current implementation is recursive. I'm having trouble determining how the recursion would have an effect in practice.

Reply:
I realized that the recursion could be needed for tuples. I redid the implementation to keep the recursion.